analyzers/tests/SonarAnalyzer.UnitTest/Rules/ValuesUselesslyIncrementedTest.cs

```diff
@@ -41,6 +41,10 @@ public class ValuesUselesslyIncrementedTest
         public void ValuesUselesslyIncremented_CSharp10() =>
             builder.AddPaths("ValuesUselesslyIncremented.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
 
+        [TestMethod]
+        public void ValuesUselesslyIncremented_CSharp11() =>
+            builder.AddPaths("ValuesUselesslyIncremented.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp11).Verify();
```

Comment:
Here we check the wrong file. It should be `ValuesUselesslyIncremented.CSharp11.cs`